1-org/.gitignore

```diff
@@ -28,6 +28,9 @@ Session.vim
 *.tfstate
 *.tfstate.*
 
+# tf lock file
+**/.terraform.lock.hcl
```

Comment:
We should allow checking in lockfiles into each stage consumer git repo. Could we create a global .gitignore at repo root so we do check it into this repo?

Reply:
Done